tests/Feature/Api/ExperienceSkillsControllerTest.php

```diff
@@ -189,6 +190,47 @@ public function testAttachSkillToApplicantDuringStoreExperienceSkill()
         ]);
     }
 
+    public function testAttachSkillToApplicantDuringBatchStore()
+    {
+        $education = factory(ExperienceEducation::class)->create();
+        $experienceSkill1 = factory(ExperienceSkill::class)->create([
+            'experience_id' => $education->id,
+            'experience_type' => 'experience_education',
+            'skill_id' => 1,
+        ]);
+        $experienceSkill2 = factory(ExperienceSkill::class)->create([
+            'experience_id' => $education->id,
+            'experience_type' => 'experience_education',
+            'skill_id' => 2,
+        ]);
+        $award = factory(ExperienceAward::class)->create();
+        $awardSkill = factory(ExperienceSkill::class)->create([
+            'experience_id' => $award->id,
+            'experience_type' => 'experience_award',
+            'skill_id' => 3,
+        ]);
+        $data = [$experienceSkill1, $experienceSkill2, $awardSkill];
+        $response = $this->actingAs($education->experienceable->user)
+            ->json('post', route('api.v1.experience-skill.store'), $data);
+        $response->assertOk();
+        $response->assertJsonFragment($experienceSkill2);
```

Comment:
@tristan-orourke Is this meant to be `$response->assertJsonFragment($experienceSkill1);`?